src/TS/map/runInfoMap.ts

```diff
@@ -1,47 +1,64 @@
 import * as L from 'leaflet';
-import { DataSetEntry } from '../models/lapDetials';
-
-export function drawLap(lapDetails: DataSetEntry[]) {
-  const latlngs: L.LatLngExpression[] = [];
-  if (!(lapDetails?.[0] || lapDetails[0]['Lat.'] || lapDetails[0]['Lon.'])) {
-    throw Error('Error loading the map');
+import { LapDetails } from '../models/lapDetials';
+export class MapStuff {
+  map;
+  constructor() {
+    this.map = L.map('map');
```

Comment:
When you install leaflet the way you did in the package.json you no longer need `L`, instead you can simply do:
```
this.map = new Map("map");
```
I think 🤔

Reply:
Oh yes, you are correct. However I imported every module from leaflet and I named that object `L` (`import * as L from 'leaflet' `). So I use that to access everything I need.